extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -967,6 +989,21 @@ export class SystemLayer {
           'background-image': `url(${control.backgroundImageUrl})`,
         });
       }
+
+      if (control.event || defaultConfig.event) {
+        const handleClick = () => {
+          this.viewerMessagingHandler_ &&
+            this.viewerMessagingHandler_.send(
+              'documentStateUpdate',
+              dict({
+                'state': AMP_STORY_PLAYER_EVENT,
+                'value': control.event || defaultConfig.event,
+              })
+            );
+        };
+
+        element.addEventListener('click', handleClick.bind(this));
```

Comment:
Can we reuse the click listener that's in `initializeListeners_`? Having two click listeners - one for buttons and one for player controls - might get confusing and harder to maintain.